linetools/analysis/absline.py

```diff
@@ -0,0 +1,67 @@
+"""
+Module related to analysis of absorption lines
+  -- Methods
+"""
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import numpy as np
+import os
+
+from astropy import units as u
+from astropy import constants as const
+
+
+# Perform AODM on the line
+def aodm(spec,idata):
+    """  AODM calculation 
+    See Savage & Sembach 1991, ApJ, 379, 245
+
+    Parameters
+    ----------
+    spec -- Tuple of (wave, fx, sig)
+    idata -- Tuple of (wrest, fval)
+
+    Returns:
+    N, sigN : float, float
+      Column and error in linear space (cm^-2)
+    flg_sat: bool
+      Set to True if saturated pixels exist
+    """
+    flg_sat = False
+
+    # Cut spectrum
+    velo,fx,sig = spec
+    wrest, fval = idata
+
+    # dv
+    delv = velo - np.roll(velo,1)
+    delv[0] = delv[1]
+
+    # Atomic data
+    cst = (const.m_e.cgs*const.c.cgs / (np.pi * 
+        (const.e.esu**2).cgs)).to(u.AA*u.s/(u.km*u.cm**2))
```

Comment:
Can you move this to the module-level instead of in this function? It can be slow.